src/logging/error/MomoError.ts

```diff
@@ -1,5 +1,8 @@
 export const momoError = {
   nonParsableInterval: new Error('non-parsable job interval'),
+  nonParsableCronSchedule: new Error('non-parsable cron schedule'),
+  missingIntervalAndCronSchedule: new Error('a job must have either an interval or a cron schedule'),
```

Comment:
lines 4 and 5 are unused

Reply:
You're right! Huh... weird - my IDE didn't highlight this for some reason.